config/config/common-config.yaml

```
@@ -656,3 +656,10 @@ networkPolicies:
 
   rookCeph:
     enabled: true
+  coredns:
+    enabled: true
+    externalDns:
+      ips:
+        - set-me
```

I would like to add our default in this one as well, just so it's very clear if you look in the defaults folder

```suggestion
    externalDns:
      ips:
        - set-me
    serviceIp:
      ips:
        - "10.233.0.3/32"
```

---

done
thx a lot